extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/exec/MSQSelectTest.java

```diff
@@ -1673,7 +1659,7 @@ public void testGroupByWithComplexColumnThrowsUnsupportedException()
         .setSql("select unique_dim1 from foo2 group by unique_dim1")
         .setQueryContext(context)
         .setExpectedExecutionErrorMatcher(CoreMatchers.allOf(
-            CoreMatchers.instanceOf(UnsupportedSQLQueryException.class),
+            CoreMatchers.instanceOf(DruidException.class),
```

Comment:
Technically various of the SQL exceptions get caught and used to tell the volcano planner that it generated a bad query and to keep looking.  None of those are converted to DruidException yet.  Also, most of the places that were throwing `QueryException` are still throwing `QueryException` because we need a release that has the `DruidException` in it before we can force those forward.

So... really, any place that potentially has other uses just didn't get converted.  We will need to go back and review them with the understanding that we now have an exception whose sole purposes is to be delivered back to the user and decide if the current handling is good.